Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -66,3 +74,24 @@ class DashboardViewModel: ObservableObject {
             .delete()
     }
 }
+
+// MARK: - Sort Option
+extension DashboardViewModel {
+    enum SortOption: Int, CaseIterable, Identifiable {
+        case oldestToNewest = 0
+        case newestToOldest = 1
+        case custom = 2
+        
+        var id: Int {
+            rawValue
+        }
+        
+        var label: String {
```

Comment:
Can you make this a `LocalizedStringResource` so that this string can be localized?
Also `SortOption` can conform to `Identifiable` to get rid of the `id: \.self` in the `ForEach` loop in `DashboardView`